netlab_testing/HTTPTests.cpp

```diff
@@ -30,7 +30,16 @@ class HTTP_Tests : public test_base {
 	netlab::HTTPServer_Impl* http_server;
 	netlab::HTTPClient_Impl* http_client;
 
-	HTTP_Tests() : test_base("", "(arp and ether src bb:bb:bb:bb:bb:bb) or (tcp port 4433 and not ether src aa:aa:aa:aa:aa:aa)")
+	HTTP_Tests() :
```

Comment:
take a look at the implementation inside test_base.
why previous implementation wasnt good?

Reply:
Fixed